doc/source/dev_guide/custom_reader.rst

```diff
@@ -431,8 +432,15 @@ On top of that, two attributes need to be defined: ``start_time`` and
             return dataset
 
         def get_area_def(self, dataset_id):
-            # TODO
-            pass
+            return pyresample.geometry.AreaDefinition(
+                "some_area_name",
+                "on-the-fly area",
+                "geos",
+                "+a=6378169.0 +h=35785831.0 +b=6356583.8 +lon_0=0 +proj=geos",
+                3636,
+                3636,
+                np.array([-5456233.41938636, -5453233.01608472, 5453233.01608472, 5456233.41938636]))
```

Comment:
It is probably best (to reduce confusion) to not have this be a numpy array. Don't worry about it right now though.

Reply:
```suggestion
                [-5456233.41938636, -5453233.01608472, 5453233.01608472, 5456233.41938636])
```